Source/WebCore/accessibility/AXTextMarker.cpp

```diff
@@ -732,20 +740,40 @@ const AXTextRuns* AXTextMarker::runs() const
     return object ? object->textRuns() : nullptr;
 }
 
-AXTextMarker AXTextMarker::findMarker(AXDirection direction, std::optional<AXID> stopAtID) const
+AXTextMarker AXTextMarker::findMarker(AXDirection direction, bool coalesceObjectBreaks, bool ignoreBRs, std::optional<AXID> stopAtID) const
```

Comment:
Rather than `bool`s, could we make these enum classes? e.g. `enum class CoalesceObjectBreaks : bool { No, Yes }`

This would help make the callsites a bit more descriptive vs. `false` / `true`.